tools/test.py

```diff
@@ -33,6 +33,18 @@ def parse_args():
         action='store_true',
         help='Whether to fuse conv and bn, this will slightly increase'
         'the inference speed')
+    group_gpus = parser.add_mutually_exclusive_group()
```

Comment:
Please specify the command used in the corresponding document.

Reply:
```python
if args.gpu_ids is not None:
    cfg.gpu_ids = args.gpu_ids
else:
    cfg.gpu_ids = range(1) if args.gpus is None else range(args.gpus)
```
We used gpus or gpu_id here